### DISCUSSION TITLE
Release a new Linux version

### DISCUSSION THREAD

**Discussion by anymor1024** [gitextensions/gitextensions#9240] [General] [answered]
GitExtensions is super perfect tools, but unfortunately the last version for Linux is 2.51 with mono which is very old.

I would like to ask you to think about Linux as well and please include in the plans the preparation of the version for Linux and to come back again with the latest version.

**Thank you very much.**

Best regards,
anymor1024

**Comment by gerhardol** [comment, marked as answer]
The mono version was dropped due to mono being limited (no Mono/WinForms updates since about 2010) and that there were no active developer for GE on mono.
The GE 2.x mono branch still exists, there has not been many contributions since last release.
If there are any developers interested in maintaining the 2.x branch, there can of course be new releases. Mono/WinForms is still a limiting factor

There has been interests in making GE cross-platform in other ways, but that will require a lot of effort. Either "someone" ports WinForms to other platforms or the GE UI layer is rewritten using something else than WinForms.
GE master now uses .NET5, so that is cross-platform.

**Reply by theoparis** [reply]
There does seem to be a new officially supported, cross platform UI framework for Dotnet 6 however it is still a preview version:

https://github.com/dotnet/maui

**Reply by gerhardol** [reply]
The same status as all other crossplatform framework: There is a considerable effort to migrate from Winforms to something else.
Maui is based on Xamarin.Forms and is supposedly limited too.

If true cross platform is happening, someone with interest need to take the lead to in the porting effort.
This will likely require more than 1000h.
